.generator/src/generator/templates/api_client.j2

```diff
@@ -7,7 +7,7 @@ require 'json'
 require 'logger'
 require 'tempfile'
 require 'time'
-require 'typhoeus'
+require 'httparty'
```

Comment:
I will look into adding this. Was previously running into some gem dependency issues

Reply:
Looked into this a bit more and at the moment adding the version information will cause circular dependency issues: https://github.com/DataDog/datadog-api-client-ruby/blob/master/datadog_api_client.gemspec#L3

I will see if I can reconstruct the gemspec file to avoid this